base/applications/mspaint/main.cpp

```diff
@@ -77,7 +77,7 @@ OFNHookProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             SendMessage(hParent, CDM_GETFILEPATH, _countof(Path), (LPARAM)Path);
             FileExtFromFilter(PathFindExtension(Path), pon->lpOFN);
             SendMessage(hParent, CDM_SETCONTROLTEXT, 0x047c, (LPARAM)PathFindFileName(Path));
```

Comment:
Side comment: we don't have a #define ... for that control ID 0x047C ?

Reply:
It's `cmb13` in `<dlgs.h>`.